### PR TITLE
minor vale related changes

### PR DESCRIPTION
**Pull request by maryamsulemani97**
This PR fixes typos and adds words to `accept.txt`

### CODE REVIEW

File: .vale/styles/style_guide/Indexing.yml [hunk 1]
```diff
@@ -0,0 +1,10 @@
+---
+# Error: style_guide.indexing.yml
+
+# Most dictionaries still list "indexation" as a term with a very specific meaning in economy so we should use “indexing”
+extends: substitution
+message: "Use '%s' instead of '%s'."
+level: warning
+swap:
+  indexation: indexing
+
```
Comment (guimachiavelli) [inline .vale/styles/style_guide/Indexing.yml:10]:
```suggestion

```

**Review by guimachiavelli — APPROVED** [review]
🥍

File: .vale/styles/style_guide/Indexing.yml (hunk 1 above)
Comment (maryamsulemani97) [reply .vale/styles/style_guide/Indexing.yml:10]: All the rules have a blank line at the end, so I guess I'll keep it 🤷🏻‍♀️

**Comment by maryamsulemani97** [comment]
bors merge

File: .vale/styles/style_guide/Indexing.yml (hunk 1 above)
Comment (guimachiavelli) [reply .vale/styles/style_guide/Indexing.yml:10]: yeah, the blank line is good, but the problem that github was complaining about was that the line had two spaces, so it wasn't technically blank.

Comment (maryamsulemani97) [reply .vale/styles/style_guide/Indexing.yml:10]: Got it, I'll fix this in another PR

**Pull request merged**